src/Presentation/Nop.Web/App_Data/Localization/Installation/installation.zhs.xml

```diff
@@ -97,40 +97,40 @@
     <Value>[推荐]</Value>
   </LocaleResource>
   <LocaleResource Name="RestartInstallation">
-    <Value>重启安装</Value>
+    <Value>重新安装</Value>

```

Comment:
I think the original translation is correct.It's really restart the application.